frontend/app/components/account-form.js

```diff
@@ -81,6 +97,7 @@ export default class AccountForm extends BaseFormComponent {
 
   @action
   setFolder(folder) {
+    this.selectedFolder = folder;
```

Comment:
huh, what's that needed for? I think you can delete the whole instance variable

Reply:
You're right.